docs/conf.py

```diff
@@ -1 +1,2 @@
 from crate.theme.rtd.conf.crate_python import *
+html_favicon = None
```

Comment:
@jodok there is already a newline character at the end of the line
python should not have a blank line at the end of the file (pep8 W391)

```console
$ pep8 docs/conf.py
docs/conf.py:4:1: W391 blank line at end of file
```